docs/src/SUMMARY.md

```diff
@@ -72,9 +72,7 @@
     - [LET-IN definitions](./lang/user/let-in.md)
     - [Higher-order operators definitions](./lang/user/higher-order-operators.md)
     - [Anonymous operator definitions](./lang/user/lambdas.md)
-    - [Recursive operator definitions](./lang/user/recursive-operators.md)
     - [Local operator definitions](./lang/user/local-operators.md)
-    - [Recursive functions](./lang/user/recursive-functions.md)
 - [Apalache operators](./lang/apalache-operators.md)
```

Comment:
Should this PR also remove these files?

Reply:
I think it's confusing if our language manual mixes TLA+ in general and its treatment by Apalache.
So far, I've read the language manual as "TLA+, as supported by Apalache".

Reply:
Interesting. We are documenting some of the operators that are not supported by Apalache. We definitely explain where Apalache is different from TLC. But I thought of it as a good javadoc-like manual for TLA+ in general, as there is no good one.

Reply:
| I do not see why we are removing these sections from the language manual

These are currently just blank files anyhow (https://apalache.informal.systems/docs/lang/user/recursive-operators.html, https://apalache.informal.systems/docs/lang/user/recursive-functions.html)

Reply:
> Interesting. We are documenting some of the operators that are not supported by Apalache. We definitely explain where Apalache is different from TLC. But I thought of it as a good javadoc-like manual for TLA+ in general, as there is no good one.

And this is definitely helpful! But maybe it should be a separate, root-level manual then, and not part of the Apalache manual? As in, if I'm reading the zsh man pages, I expect them to document how zsh works, not POSIX shell at large?